app/controllers/fileUpload/ProcessingRequestController.scala

```diff
@@ -46,11 +46,11 @@ class ProcessingRequestController @Inject()(
       parsingAndValidationOutcomeCacheConnector.getOutcome.flatMap {
         case Some(ParsingAndValidationOutcome(Success, _, _)) =>
           Future.successful(Redirect(controllers.fileUpload.routes.FileUploadSuccessController.onPageLoad(waypoints, eventType)))
-        case Some(ParsingAndValidationOutcome(GeneralError, _, _)) =>
+        case Some(ParsingAndValidationOutcome(GeneralError, js, _)) =>
```

Comment:
Do we actually use the `js` down the line?

Reply:
Sorry, no, I think I was just testing something - I'll take it out